src/Resources/views/CRUD/Association/edit_many_script.html.twig

```diff
@@ -175,7 +175,7 @@ This code manages the many-to-[one|many] association field popup
 
                 // capture the submit event to make an ajax call, ie : POST data to the
                 // related create admin
-                jQuery('a', field_dialog_{{ id }}).on('click', field_dialog_form_action_{{ id }});
+                jQuery(document).on('click','#field_dialog_{{ id }} a', field_dialog_form_action_{{ id }});
                 jQuery('form', field_dialog_{{ id }}).on('submit', field_dialog_form_action_{{ id }});
```

Comment:
dont you need to do the same with this other bind?

Reply:
Not really, you can click any link inside of a modal and it will refresh the modal but you can submit only 1 time and when you do that the modal is closed

Reply:
I mean, I can do it but no benefit really.

Reply:
So, the form tag is not replaced when the form is reloaded

Reply:
Will test it later today but I did not have a problem with submitting at all.

Reply:
Not needed for submit.